apps/package.json

```diff
@@ -27,7 +27,10 @@
   },
   "browserify-global-shim": {
     "react": "React",
-    "react-dom": "ReactDOM"
+    "react-dom": "ReactDOM",
+    "johnny-five": "JohnnyFive",
+    "playground-io": "PlaygroundIO",
+    "chrome-serialport": "ChromeSerialport"
```

Comment:
I suspect these better belong in browserify-shim rather than as global-shims. The only time we should need to have packages in global shim is if we're requiring other packages that require them (i.e. we require a color-picker module that requires React, thus it needs to be in global).

Not sure there's any harm in this, but probably better off putting these in browserify-shim.